docker-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pandas==1.0.3
+xlrd==1.2.0
+sqlalchemy==1.3.19
```

Comment:
It'd probably be best to add `psycopg2-binary` in here and do away with `requirements.txt` altogether (it isn't used after dockerization).